src/schema/v2/schema.ts

```diff
@@ -123,6 +124,11 @@ const rootFields = {
   geneFamiliesConnection: GeneFamilies,
   highlights: HighlightsField,
   homePage: HomePage,
+  _do_not_use_image: {
```

Comment:
I'm obvs missing product context but wondering whats up with this naming?

Reply:
It's so that you don't use it, that's why it's named that 😆 

It's only used for internally, to stitch in image data from an upstream schema, but resolve the special Metaphysics `Image` type.